en_US/deploy/cluster/create-cluster.md

```diff
@@ -516,3 +516,30 @@ To use TCP IPv4 and TCP IPv6, you can set with the `cluster.proto_dist` in `emqx
 To enable SSL, you first need to set the `cluster.proto_dist` to `inet_tls`, then configure the `ssl_dist.conf` file in the `etc` folder and specify the TLS certificate. For details, see [Using TLS for Erlang Distribution](https://www.erlang.org/doc/apps/ssl/ssl_distribution.html).
 
 <!--need an example code here-->
+
+## Pseudo-Distributed Cluster
+
+EMQX also provides a pseudo-distributed cluster feature for testing and development purposes. It refers to a cluster setup where multiple instances of EMQX are running on a single machine, with each instance configured as a node in the cluster.
+
+After starting the first node, use the following command to start the second node and join the cluster manually. To avoid port conflicts, we need to adjust some listening ports:
+
+```bash
+EMQX_NODE__NAME='emqx2@127.0.0.1' \
+    EMQX_LOG__FILE_HANDLERS__DEFAULT__FILE='log2/emqx.log' \
+    EMQX_STATSD__SERVER='127.0.0.1:8124' \
+    EMQX_LISTENERS__TCP__DEFAULT__BIND='0.0.0.0:1882' \
+    EMQX_LISTENERS__SSL__DEFAULT__BIND='0.0.0.0:8882' \
+    EMQX_LISTENERS__WS__DEFAULT__BIND='0.0.0.0:8082' \
+    EMQX_LISTENERS__WSS__DEFAULT__BIND='0.0.0.0:8085' \
+    EMQX_DASHBOARD__LISTENERS__HTTP__BIND='0.0.0.0:18082' \
+    EMQX_NODE__DATA_DIR="./data2" \
+./bin/emqx start
+
+./bin/emqx ctl cluster join emqx1@127.0.0.1
+```
+
+The above code example is to create a cluster manually, you can also refer to the [auto clustering](./create-cluster.md#auto-clustering) section on how to create a cluster automatically.
+
+The dashboard is designed under the assumption that all cluster nodes use the same port number. Using distinct ports on a single computer may cause Dashboard UI issues, therefore, it is not recommended in production.
```

Comment:
Not sure what does this statement mean exactly. We explicitly set different port numbers on different nodes above.